lib/backend/syncersv1/felixsyncer/felixsyncerv1.go

```diff
@@ -72,9 +73,11 @@ func New(client api.Client, callbacks api.SyncerCallbacks) api.Syncer {
 			ListInterface:   model.ResourceListOptions{Kind: apiv3.KindHostEndpoint},
 			UpdateProcessor: updateprocessors.NewHostEndpointUpdateProcessor(),
 		},
-		{
-			ListInterface: model.BlockListOptions{},
-		},
+	}
+
+	// If using Calico IPAM, include IPAM resources the felix cares about.
```

Comment:
We could alternatively implement a dummy version of IPAM blocks so that we don't need this extra plumbing. The main thing is that Felix tries to list/watch IPAM blocks and those don't exist when using host-local. 

I don't have a strong preference on how we do this - @fasaxc WDYT?